plugins/network-elements/nuage-vsp/src/com/cloud/network/resource/NuageVspResource.java

```diff
@@ -270,15 +258,15 @@ public PingCommand getCurrentStatus(long id) {
             _shouldAudit = true;
             return null;
         }
-        if (_cmsUserInfo == null || _cmsUserInfo.length < 2) {
+        if (_cmsUserLogin == null || _cmsUserLogin.isEmpty() || _cmsUserPassword == null || _cmsUserPassword.isEmpty()) {
```

Comment:
@nlivens you can use StringUtils.isEmpty(CharString cs) [https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html] in order to check if a String is empty or null.